projects/static/projects/js/members.js

```diff
@@ -0,0 +1,28 @@
+function resetRole(selector_id) {
+  var selector = document.getElementById(selector_id);
+  for (var j = 0; j < selector.options.length; j++) {
+    var opt = selector.options[j];
+    if (opt.disabled) {
```

Comment:
In front-end land, it is considered a bit of a no-no to do this, relying on presentation of the UI as a holder of state. JS has data attributes now, which you could use to store the initial/current role state in order to reset it.

```html
<select data-initial="member">
  <option value="member">Member</option>
</select>
```

```js
document.querySelect('select')[0].dataset.initial
```

The reason for watching out for storing some "state" in the disabled attribute is, what if some other UI element decides to disable this later for whatever reason? Or what if there is some other weird browser thing that temporarily disables the input? There are tons of ways an element could get disabled, not all of which (indeed, not most of which) are under this code's control.